packages/p2p/src/components/my-ads/create-ad-add-payment-method-modal.jsx

```diff
@@ -63,7 +63,7 @@ const CreateAdAddPaymentMethodModal = () => {
             is_open={my_ads_store.should_show_add_payment_method_modal}
             title={localize('Choose payment method')}
         >
-            <Modal.Body>
+            <Modal.Body className={my_profile_store.selected_payment_method && 'p2p-my-ads__modal-body--scroll'}>
```

Comment:
Why do we need this when we can use `classNames` here?

Reply:
Since it was a single class added conditionally, I went with this approach.